examples/style_sheets/plot_ggplot.py

```diff
@@ -6,8 +6,8 @@
 
 .. [1] http://www.huyng.com/posts/sane-color-scheme-for-matplotlib/
 
-.. _ggplot: http://had.co.nz/ggplot/
-.. _R: http://www.r-project.org/
+.. _ggplot: https://github.com/yhat/ggplot
```

Comment:
sorry, my bad. fixed to http://ggplot2.org/  I'm not sure about the `2`, though. But it looks right.